spec/spec_helper.rb

```diff
@@ -48,7 +51,7 @@
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
 # The settings below are suggested to provide a good initial experience
-# with RSpec, but feel free to customize to your heart's content.
+# with RSpec, but feel free to customize to your heart"s content.
```

Comment:
Nice @chrismitchell indeed I thought about opening a different PR for that, I'll do it. 😄